astropy/convolution/tests/test_kernel_class.py

```diff
@@ -2,21 +2,19 @@
 
 import itertools
 
-import pytest
 import numpy as np
-from numpy.testing import assert_almost_equal, assert_allclose
-
-from ..convolve import convolve, convolve_fft
-from ..kernels import (
-    Gaussian1DKernel, Gaussian2DKernel, Box1DKernel, Box2DKernel,
-    Trapezoid1DKernel, TrapezoidDisk2DKernel, MexicanHat1DKernel,
-    Tophat2DKernel, MexicanHat2DKernel, AiryDisk2DKernel, Ring2DKernel,
-    CustomKernel, Model1DKernel, Model2DKernel, Kernel1D, Kernel2D)
+import pytest
+from numpy.testing import assert_allclose, assert_almost_equal
 
 from ..utils import KernelSizeError
+from ..kernels import (Kernel1D, Kernel2D, Box1DKernel, Box2DKernel, CustomKernel,
```

Comment:
this must be beyond 80 characters while most of this file is within

Reply:
I set the line limit to 99, does astropy use 80 everywhere?

Reply:
it depends on the module, some are 80 some are 100.

Reply:
in fact all the other tools in `setup.cfg` are configured to 100 lines.

Reply:
http://docs.astropy.org/en/stable/development/codeguide.html#coding-style-conventions

It's set to 100 so it won't complain for modules where the limit is not 80.

Reply:
yeah it's one or the other ;)

Reply:
We had this discussion quite a while back, and it was decided one should follow the style of the module - PEP8 states it should be 80, so I think 100 should be the exception, not the rule.

Reply:
Why not 79? We need that 80th char for `\0`. 😄